File: src/Tawtheef.Websites/operations-web/src/app/pages/user/employee/exams-management/models/exam-job.dto.ts
import { ExamLookupItemDto } from "./exam-lookups.dto";

export interface ExamJobDto extends ExamLookupItemDto {
  jobNumber: string;
  managementAr: string | null;
  managementEn: string | null;
  departmentAr: string | null;
  departmentEn: string | null;
  specializationsAr: string[];
  specializationsEn: string[];
}

File: src/Application.Operation/Features/Employee/Exams/DTOs/ExamJobDto.cs
namespace Application.Operation.Features.Employee.Exams.DTOs;

public sealed record ExamJobDto(Guid Id, string NameAr, string NameEn, string JobNumber,
    string? ManagementAr, string? ManagementEn, string? DepartmentAr, string? DepartmentEn,
    List<string> SpecializationsAr, List<string> SpecializationsEn);
